addons/mail/static/src/models/call_view.js

```diff
@@ -80,7 +80,7 @@ registerModel({
          * @private
          */
         _computeCallSideBarView() {
-            if (this.mainParticipantCard && this.hasSidebar && !this.threadView.compact) {
+            if (this.activeRtcSession && this.isSidebarOpen && !this.threadView.compact) {
```

Comment:
be careful that if `hasSidebarButton` becomes false is doesn't necessarily reset the `isSidebarOpen` selection, it's already there in previous code but it might be strange